web/static/js/components/questionnaires/QuotaCompletedMsg.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { Component, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import { InputWithLabel, Card } from '../ui'
+import * as actions from '../../actions/questionnaire'
+
+class QuotaCompletedMsg extends Component {
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired,
+    questionnaire: PropTypes.object
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      editing: false
+    }
+    this.quotaCompletedMsgChanged = this.quotaCompletedMsgChanged.bind(this)
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick() {
+    this.setState({editing: !this.state.editing})
+  }
+
+  quotaCompletedMsgChanged(e, mode) {
+    const { dispatch } = this.props
+    e.preventDefault()
+    switch (mode) {
+      case 'sms':
+        dispatch(actions.setSmsQuotaCompletedMsg(e.target.value))
+        break
+      case 'ivr':
+        dispatch(actions.setIvrQuotaCompletedMsg(e.target.value))
+        break
+    }
+  }
+
+  render() {
+    const { questionnaire } = this.props
+
+    let quotaCompletedMsgs = questionnaire.modes.map((mode) => {
+      const value = questionnaire.quotaCompletedMsg && questionnaire.quotaCompletedMsg[questionnaire.defaultLanguage][mode] ? questionnaire.quotaCompletedMsg[questionnaire.defaultLanguage][mode] : ''
+
+      return (
+        <div className='row' key={`quota-completed-${mode}`}>
+          <div className='input-field'>
+            <InputWithLabel value={value} label={`${mode == 'sms' ? 'SMS' : 'Phone'} message`}>
```

Comment:
`mode == 'sms' ? 'SMS' : 'Phone'` is bound to bring problems if a third mode is added later ;-)